### DISCUSSION TITLE
Understanding the "note" feature

### DISCUSSION THREAD

**Discussion by thiswillbeyourgithub** [Medvedev91/timeto.me#68] [Q&A]
Hi,

[In an issue](https://github.com/Medvedev91/timeto.me/issues/63#issuecomment-2226430286) I learned that : 
>  >   Addendum: I have yet to understand what notes are for!

> I use it to keep simple data, like wifi passwords.

Just to make sure I got that right: the "note" feature has nothing to do with timetracking and is just to store text data, and is not related to any activity in particular or any other menu right?

Thanks!

**Comment by Medvedev91** [comment]
Yes, that's right. Someday I'll add this to the readme.

**Comment by ericpso** [comment]
Ah, I was hoping I could add a note to a task or shortcut for planning.

I would like to add a note to a task that has a "Map Navigation" shortcut so that I could easily copy an address and paste it on the navigation app.

**Reply by Medvedev91** [reply]
> Ah, I was hoping I could add a note to a task or shortcut for planning.

Do you mean like this https://github.com/Medvedev91/timeto.me/issues/200 and this https://github.com/Medvedev91/timeto.me/issues/125 or Note as feature?

> I would like to add a note to a task that has a "Map Navigation" shortcut

It's always the same address or not? If the same I guess it is possible to create a shortcut with deep link payload for navigation app.
